app/main/helpers/validation.py

```diff
@@ -282,7 +283,7 @@ def errors(self):
         return errors_map
 
 
-def is_valid_percentage(value):
+def is_valid_percentage(value: Any) -> bool:
```

Comment:
Would `(value: Union[str, int]) -> bool` work?

Reply:
That would work for the function as currently used, but I'm not sure `Any` is wrong. `is_valid_percentage` will happily accept and correctly process a value of any type.